### PR TITLE
feat: add context menu option; add manifest icons.

### PR DESCRIPTION
**Pull request by symant233**
### 1. add manifest icons: 
image
### 2. add context menu option: 
image
After click context menu option, `PopupCard` will show with empty text on the place where mouse right clicked.
image

### CODE REVIEW

**Comment by symant233** [comment]
update: [hide translated area when the text is empty](https://github.com/yetone/openai-translator/pull/128/commits/357c455f2cb5e7f61f593be983de5f3f77ff849d)
image

**Comment by yetone** [comment]
Great job! Thanks!

File: src/content_script/index.tsx [hunk 1]
```diff
@@ -167,7 +167,10 @@ async function showPopupThumb(text: string, x: number, y: number) {
 }
 
 async function main() {
+    let lastMouseEvent: MouseEvent
```
Comment (yetone) [inline src/content_script/index.tsx:170]:
```suggestion
    let lastMouseEvent: MouseEvent | undefined
```

**Comment by yetone** [comment]
Suddenly remembered there is a similar PR here, why don't you guys discuss how to turn it into one?

https://github.com/yetone/openai-translator/pull/82

**Comment by symant233** [comment]
Ok, I will check if these two PRs have conflict.

**Comment by pengx17** [comment]
@yetone I went over the code by @symant233. her/his solution requires less permission than mine and I think we shall accept this PR instead of mine. Will close mine now.

**Comment by symant233** [comment]
I have implemented a part of the feature based on your PR. Can I have your email so that I can add you as a co-author? @pengx17

**Comment by pengx17** [comment]
sure, my email is pengxiao at outlook dot com

**Comment by symant233** [comment]
All features have been tested and are ready to merging. @yetone

**Comment by yetone** [comment]
@pengx17 Thank you very much!

**Comment by yetone** [comment]
@symant233 Thank you very much, just need to resolve the conflict.

**Comment by symant233** [comment]
I have resolved the conflict. @yetone 
During testing, I found another problem which is not related to this PR that the icon reference error on the browser page, which may require further fixing.
image

**Comment by yetone** [comment]
@symant233 Thank you for helping me find the problem. It was caused by me and I have already fixed it. Thank you very much.

**Pull request merged**